src/deploy/AWSConfig.js

```diff
@@ -113,6 +128,15 @@ export default class AWSConfig {
         type: 'string',
         default: DEFAULT_LAMBDA_FORMAT,
       })
+      .option('aws-create-authorizer', {
+        // eslint-disable-next-line no-template-curly-in-string
+        description: 'Authorizer to create or update during linking. The string can contain placeholders (note that all dots (\'.\'). Example "helix-authorizer_${version}".',
```

Comment:
Maybe it's just me or the escaping:
> note that all dots (\'.\'). 

a verb would possibly help ;-) does that mean all dots must be escaped?

> Example "helix-authorizer_${version}".

Is the last dot part of the example, or a sentence terminator? In the latter case, I'd skip it.